docs/onboarding-guide.md

```diff
@@ -0,0 +1,171 @@
+
+
+
+### [AWS.Deploy.Recipes](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes)
+
+This component contains the recipes that serve as the backbone of each deployment recommendation. It also contains the CDK templates that is used to deploy the customer’s .NET application as a CloudFormation stack
+
+#### Key Components
+
+* **[RecipeDefinitions](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/RecipeDefinitions)** - It contains the deployment recipes that target different AWS services. Each recipe file contains a list of option settings which is a collection of AWS resources that users can customize as per their needs. 
+    * All deployment recipes are governed by the same schema found [here](https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.Recipes/RecipeDefinitions/aws-deploy-recipe-schema.json).
+    * See [here](https://aws.github.io/aws-dotnet-deploy/docs/support/) to look at the support matrix between recipes and different .NET application types.
+* [**DeploymentBundleDefinitions**](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/DeploymentBundleDefinitions) - It contains the common option settings that users can customize for container based and non-container based recipes. All recipes have a [DeploymentBundle](https://github.com/aws/aws-dotnet-deploy/blob/66fb1e34bbea10bbd995596e1eec801b79ff9102/src/AWS.Deploy.Recipes/RecipeDefinitions/aws-deploy-recipe-schema.json#L59) property that specifies how to package the user’s .NET project for deployment. 
+    * *Container* - Indicates that Docker is used to build the Docker image and pushed to Amazon Elastic Container Registry.
+    * *DotnetPublishZipFile -* Indicates that the `dotnet publish`  is used to prepare the user’s .NET project for deployment.
+* [**CdkTemplates**](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/CdkTemplates) - It contains the CDK templates for recipe files found [here](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/RecipeDefinitions). All recipes with the [DeploymentType](https://github.com/aws/aws-dotnet-deploy/blob/66fb1e34bbea10bbd995596e1eec801b79ff9102/src/AWS.Deploy.Recipes/RecipeDefinitions/aws-deploy-recipe-schema.json#L53) set to *“CdkProject”* have their own specific CDK template identified by the [CdkProjectTemplate](https://github.com/aws/aws-dotnet-deploy/blob/66fb1e34bbea10bbd995596e1eec801b79ff9102/src/AWS.Deploy.Recipes/RecipeDefinitions/aws-deploy-recipe-schema.json#L65) property.
+
+
+
+### **[AWS.Deploy.CDK.Common](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes.CDK.Common)**
+
+This component contains the common utility classes and methods that are used by the CDK templates found [here](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/CdkTemplates).
```

Comment:
We should call out that we don't take references on this project in the CLI components to avoid the 70 Meg CDK NuGet dependency. If we need code from this project we directly link the code files into the project like we do for the orchestrator. https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.Orchestration/AWS.Deploy.Orchestration.csproj#L11

Reply:
Done